.github/mentor-guidelines.md

```diff
@@ -0,0 +1,60 @@
+
+O guia não é uma regra que deve ser seguida, afinal cada pessoa consegue mentorar de uma maneira visto seu modo como se organiza ou mesmo relacionado a sua experiência mentorando. Portanto devemos tentar ao máximo seguir este guia somente onde temos uma observação de **necessário***, o restante pode ser ignorado caso você tenha uma forma de mentorar que seja efetiva.
+
+Caso você tenha um modelo de mentoria que vai utilizar, por favor, escreva um artigo para adicionarmos aqui para guiar outras pessoas que estejam entrando no Mentoria.
+
+## Processo de mentoria
+
+O processo de mentoria consiste em ajudarmos a pessoa a conquistar seus objetivos, mas não necessáriamente em ensinarmos tudo para elas.
+
+Caso venhamos a sentir que é necessário ensinar a pessoa, então podemos fazer isso conforme nosso tempo livre, mas, na maioria dos casos, enviar links, vídeos e tutoriais para as pessoas já os ajuda.
+
+A grande dificuldade das pessoas está em criar metas e seguir um plano para conquistá-las. Estamos aqui para ajudar a criar essa maturidade pessoal. :muscle:
+
+Durante o processo de mentoria executamos alguns passos padronizados:
+
+1. Entrevista inicial
+1. Explicação de como funciona uma mentoria
+1. Início do trabalho de criação de metas
+1. Apoio para execução das ações que direcionam às metas
+1. Trabalho de conclusão de Mentoria (TCM)
+
+Vamos entender cada passo.
+
+### Entrevista inicial
+
+A entrevista inicial é algo muito importante no processo de mentoria, pois iremos identificar as necessidades de quem está nos contatando.
+
+Nesta entrevista devemos questionar a pessoa afim de conseguir informações como:
+
+- Qual seu sonho de carreira?
+- Você realmente conhece essa área de atuação?
+- Por que você se interessa por essa área?
+- Você conhece mais alguém que trabalha na área?
+- Você se dispõe a estudar e praticar conforme eu lhe orientar?
+- Você acredita que podemos criar um plano para o seu futuro profissional em 2 meses?
+- O que você já sabe sobre essa área de atuação?
+- Como você estuda algo novo?
+- Como você pratica algo novo?
+- Você gosta de estudar e praticar?
+
+### Explicação de como funciona uma mentoria
+
+Muita gente não entende o que é ou como funciona o processo de mentoria, mesmo lendo nossa documentação e isso é extremamente comum.
+
+Nesse primeiro momento também paramos para explicar para a pessoa como funciona o processo de mentoria baseado em nossa documentação ([aqui](https://github.com/training-center/mentoria/)) e em nossas reuniões de mentores(as).
+
+Também é nesse passo em que nós devemos enviar o formulário para novos(as) mentorados(as) para cadastro dessa pessoa.
+
+O formulário é [esse aqui](https://goo.gl/forms/F7kBUdLPpIml1yCa2) e é muito importante para conseguirmos enviar vagas para essas pessoas, assim como projetos para praticar.
+
+### Início do trabalho de criação de metas
```

Comment:
Nao sei se pode ajudar nessa parte, mas o @regishideki tem um framework de estudos baseado em PDCA que é bem legal. 
Você pode ver mais sobre isso nesse vídeo e em 2 blogposts do diário de bordo da Creditas.

https://youtu.be/Hy0jKIGbQrM
http://diariodebordo.creditas.com.br/framework-de-estudos-introducao/
https://diariodebordo.creditas.com.br/framework-de-estudos-planejar/

Reply:
Opa! Vai ajudar sim, @camilacampos 

Valeeeeeeeus!!!